addon/mixins/run.js

```diff
@@ -239,7 +243,13 @@ export default Mixin.create({
     assert(`Called \`this.throttleTask('${name}', ${timeout})\` where 'this.${name}' is not a function.`, typeof this[name] === 'function');
     assert(`Called \`throttleTask\` on destroyed object: ${this}.`, !this.isDestroyed);
 
-    run.throttle(this, name, timeout);
+    let pendingThrottles = getOrAllocate(this, '_pendingThrottles', Array);
```

Comment:
Can we add `_pendingThrottles` to init similar to `this._pendingDebounces = undefined;` on L46

Reply:
Done.